valhalla/thor/worker.h

```diff
@@ -69,24 +75,30 @@ class thor_worker_t : public service_worker_t {
                                           const Location& origin,
                                           const Location& destination);
   void route_match(Api& request);
-  std::vector<std::tuple<float, float, std::vector<thor::MatchResult>>> map_match(Api& request);
-  void path_map_match(const std::vector<meili::MatchResult>& match_results,
-                      const std::vector<PathInfo>& path_edges,
-                      TripLeg& leg,
-                      std::unordered_map<size_t, std::pair<RouteDiscontinuity, RouteDiscontinuity>>&
-                          route_discontinuities);
+  std::vector<std::tuple<float, float, std::vector<meili::MatchResult>>> map_match(Api& request);
```

Comment:
For some of these composite types could have type aliases (either using `typedef` or `using`) that clarify what they actually mean. What are these floats supposed to be?

```
std::vector<std::tuple<float, float, std::vector<meili::MatchResult>>> 
```

Same thing goes for the `std::vector` and `std::deque` in the new declarations below.

Reply:
ive documented this return type locally. ill push it up shortly